pkg/collector/python/datadog_agent.go

```diff
@@ -9,6 +9,7 @@ package python
 
 import (
 	"context"
+	"github.com/mailru/easyjson/jlexer"
```

Comment:
Third party imports should be in second block of imports (see where `yaml` is imported), not in the stdlib import block

Reply:
Ah sorry, most of these imports are from my auto-import, I'll get these fixed to follow the convention.